effeciency testing.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+import math
+EffMap = pd.read_csv('analysis/Eff228.csv')
+
+def power_output(map, torque, w, angular_velocity = True):
+    """
+
+    :param map: csv map for torque vs rpm
+    :param torque: torque input
+    :param w: angular velocity input
+    :param angular_velocity: set to false if input for w is in units of RPM instead of angular velocity
+    :return: power output given torque and angular velocity input
+    """
+    torqueprox = round(torque)
+    if angular_velocity:
```

Comment:
Bad variable naming - first thought when I see angular velocity is that it expresses the actual quantity in angular vel, not a boolean. Just constrain the function input to standard metric variables (angular vel in this case).

Reply:
Agreed